Multisig/UI/UI Library/Views/TableViewCell/ReviewTransactionCells/ValueChangeTableViewCell.swift

```diff
@@ -21,9 +22,17 @@ class ValueChangeTableViewCell: UITableViewCell {
         valueAfter.setStyle(.primary)
     }
 
-    func set(title: String, valueBefore: String, valueAfter: String) {
+    func setBeforeAfter(title: String, valueBefore: String, valueAfter: String) {
```

Comment:
I would make these two setters as one method and adjust accordingly

Reply:
Or you can use DetailExpandableTextCell which can achieve same results

Reply:
I don't understand. What do you mean by using one setter and adjust accordingly? 
What should be the parameters of that one method? 
```
title, value, valueBefore, valueAfter?
```
I would prefer to rename ValueChangeTableViewCell to NamedValueTableViewCell. And keep both methods.

Reply:
Maybe `ChangingValueTableViewCell`?

Reply:
I sill go with the DetailExpandableTextCell.

Reply:
I am unable to make the DetailExpandableTextCell look like the ValueChangeTableViewCell

Reply:
For reference, see how it's done in TransactionDetailCellBuilder line 539, isn't it the same what we want here ?

Reply:
Are you referring to this? 
![image](https://user-images.githubusercontent.com/246473/166917176-fd096e2d-4d5f-448e-b162-5d18256561b1.png)

Reply:
Like this?

https://user-images.githubusercontent.com/246473/166917979-65bff0ce-73a4-4b79-b2ef-bcc204a73484.mov

Reply:
Not the same: 
![image](https://user-images.githubusercontent.com/246473/166922752-c111e0e8-105b-4032-8d7b-7b34dbb24b0f.png)

Reply:
Will revert